lib/WsdlParserCommon.js

```diff
@@ -62,7 +62,7 @@ function getNodeByName(parentNode, prefix, nodeName) {
  */
 function getNodeByQNameLocal(parentNode, nodeName) {
 
-  let key = Object.keys(parentNode).find((key) => {
+  let key = Object.keys(parentNode).filter((key) => {
```

Comment:
While we were using the find method 'key' was receiving a string value, but after using the filter method it is getting an array value.
In `return parentNode[key]` from this function you still use the key variable as a string value, isn't?

Reply:
indeed i modified this function prior to create the new one getNodeByQNameLocalArray and then i did not return the original code.